app/extensions/brave/content/scripts/pageInformation.js

```diff
@@ -130,7 +130,7 @@
   if (window.top !== window.self) return
 
   // Don't allow ledger to run in incognito
-  if (chrome.extension.inIncognitoContext) {
+  if (chrome.extension.inIncognitoContext || chrome.contentSettings.torEnabled == 'block') {
```

Comment:
Maybe I don't understand what chrome.contentSettings.torEnabled means?

Reply:
@riastradh-brave `chrome.contentSettings.torEnabled == 'block'` means this is running in a tor tab. sorry slightly confusing but the chromium contentsettings values can only be the strings 'block' and 'allow'.